cmd/root.go

```diff
@@ -66,70 +59,40 @@ var rootCmd = &cobra.Command{
 	},
 }
 
+// check if the version file exists in the container
+func checkVersionExists(versionUrl string) bool {
+	resp, err := http.Get(versionUrl)
+	if err != nil {
+		log.Err("checkVersionExists: error getting version file from container [%s]", err)
+		return false
+	}
+
+	return (resp.StatusCode >= 200 && resp.StatusCode <= 202)
```

Comment:
Shall we explicitly check for 404 to say file exists or not?

Reply:
Using 404 status code to check the existence of the file